package.json

```diff
@@ -25,45 +25,45 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@react-native-async-storage/async-storage": "^1.16.1",
-    "@react-native-community/masked-view": "^0.1.11",
-    "@react-navigation/bottom-tabs": "^6.2.0",
-    "@react-navigation/native": "^6.0.8",
-    "@react-navigation/native-stack": "^6.5.0",
-    "axios": "^0.26.0",
+    "@react-native-async-storage/async-storage": "^1.17.10",
+    "@react-native-masked-view/masked-view": "^0.2.7",
```

Comment:
I think we don't need this dependency anymore. It used to be required by `@react-navigation` but I think it's not anymore